include/sway/config.h

```diff
@@ -507,7 +521,8 @@ struct sway_config {
 	list_t *command_policies;
 	list_t *feature_policies;
 	list_t *ipc_policies;
-
+	
+	struct libtouch_engine *gesture_engine;
```

Comment:
I'm not seeing where this is free'd/deinitialized. Also, is the engine reusable between config reloads? If so, it should be moved to `sway_server`. If not, it is fine here.